src/hgraph/test/_node_printer.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, start: bool = True, eval: bool = True, stop: bool = True, nod
         self.graph = graph
 
     def _print(self, evaluation_clock: "EvaluationClock", msg: str) -> None:
-        print(f"[{datetime.utcnow()}][{evaluation_clock.evaluation_time}] {msg}")
+        print(f"[{datetime.now(UTC)}][{evaluation_clock.evaluation_time}] {msg}")
```

Comment:
Why the change here?

Reply:
Python 3.12 deprecates the `utcnow` function